testnet/stacks-node/src/burnchains/mock_events.rs

```diff
@@ -235,9 +245,10 @@ impl MockController {
 
         *next_burn_block += 1;
 
-        info!("Layer 1 block mined");
+        info!("Layer 1 block mined, {:?}", &new_block);
```

Comment:
Can you change this to a k/v style log?

Something like:

```suggestion
        info!("Layer 1 block mined"; "block_height" => new_block.block_height, "index_block_hash" => %new_block.index_block_hash, ...);
```

Reply:
used k/v logs throughout